README.md

```diff
@@ -1 +1,188 @@
+
+> Describe between 1-3 key features of the application.
+- **Linters SetUp**
+- **Follow HTML and CSS best practice**
+- **Display Text**
+- **Follow GitHub flow**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://qusai122.github.io)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+
+<!--
+Example commands:
+```sh
+  cd my-folder
+  git clone git@github.com:KUBAHO3/Hello-Microverse.git
+```
+--->
+### Install
+Install this project with:
+npm
+Example command:
+```sh
+  cd my-project
+  npm init -y
+```
+### Usage
+To run the project, execute the following command:
+npm start or live server
+Example command:
+```sh
+  GitHub Pages Server
+```
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+### Deployment
+You can deploy this project using:
+GitHub Pages
+but it's not done yet
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: **Qusai Issa**
+- GitHub: [@qusai122](https://github.com/qusai122)
+- LinkedIn: [Qusai Issa](https://www.linkedin.com/in/qusai-issa-ab5a59156/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## :telescope: Future Features <a name="future-features"></a>
+- [ ] **making it responsive**
+- [ ] **Adding Animation**
+- [ ] **Making it dynamic**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [https://github.com/qusai122/](../../issues/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## :star:️ Show your support <a name="support"></a>
+If you like this project please follow me on github & twitter and also connect on Linkedin.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse for this exercise.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FAQ (optional) -->
+## :question: FAQ (OPTIONAL) <a name="faq"></a>
+- **[Question_1]**
+  - [Answer_1]
+- **[Question_2]**
+  - [Answer_2]
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The link to your license is still broken. Kindly adjust the paths to look like  `[MIT](LICENSE.md)` in your readme file so that it does not raise a `404` page error